uniffi_macros/src/lib.rs

```diff
@@ -61,15 +63,24 @@ pub fn build_foreign_language_testcases(paths: proc_macro::TokenStream) -> proc_
 
 /// Newtype to simplifying parsing a list of file paths from macro input.
 #[derive(Debug)]
-struct FilePaths(Vec<String>);
+struct FilePaths {
+    idl_file: String,
+    test_scripts: Vec<String>,
+}
 
 impl syn::parse::Parse for FilePaths {
     fn parse(input: syn::parse::ParseStream) -> syn::Result<Self> {
-        Ok(FilePaths(
-            Punctuated::<LitStr, Token![,]>::parse_terminated(input)?
-                .iter()
-                .map(|s| s.value())
-                .collect(),
-        ))
+        let idl_file: LitStr = input.parse()?;
+        let _comma: Token![,] = input.parse()?;
+        let array_contents;
+        bracketed!(array_contents in input);
+        let test_scripts = Punctuated::<LitStr, Token![,]>::parse_terminated(&array_contents)?
+            .iter()
+            .map(|s| s.value())
+            .collect();
+        Ok(FilePaths {
+            idl_file: idl_file.value(),
+            test_scripts,
+        })
```

Comment:
There might be simpler way of doing this 😅

Reply:
We might be able to come up with something more declarative as we learn more, but this LGTM!